### DISCUSSION TITLE
Does capacitor support variables for podspec tag?

### DISCUSSION THREAD

**Discussion by mic345** [ionic-team/capacitor#3538] [Q&A] [answered]
Hi there,

Does capacitor support variables for podspec tag? I'm asking since we tried to install the [official ionic firebase analytics](https://ionicframework.com/docs/native/firebase-analytics) plugin and get [the same error as this guy](https://github.com/chemerisuk/cordova-plugin-firebase-analytics/issues/166) (actually there are [several of those](https://github.com/chemerisuk/cordova-plugin-firebase-analytics/issues?q=is%3Aissue+capacitor)). The plugin developer thinks it's related to capacitor's variables support for podspec tags.

Thanks for the help!
mic.

**Comment by jcesarmobile** [comment, marked as answer]
On 2.4.1 it will replace the variable with the default value, but you can’t configure a specific version
https://github.com/ionic-team/capacitor/pull/3525

**Comment by mic345** [comment]
Sounds good! Many thanks for the help :-)
